src/core/lambda/io.rs

```diff
@@ -72,16 +73,16 @@ impl IO {
     ) -> Pin<Box<dyn Future<Output = Result<ConstValue, EvaluationError>> + 'a + Send>> {
         Box::pin(async move {
             match self {
-                IO::Http { req_template, dl_id, .. } => {
+                IO::Http { req_template, dl_id, http_filter, .. } => {
                     let req = req_template.to_request(&ctx)?;
                     let is_get = req.method() == reqwest::Method::GET;
 
                     let res = if is_get && ctx.request_ctx.is_batching_enabled() {
                         let data_loader: Option<&DataLoader<DataLoaderRequest, HttpDataLoader>> =
                             dl_id.and_then(|index| ctx.request_ctx.http_data_loaders.get(index.0));
-                        execute_request_with_dl(&ctx, req, data_loader).await?
+                        execute_request_with_dl(&ctx, req, data_loader, http_filter.clone()).await?
```

Comment:
This is hot path. Let's not perform any clones here. Pass the filter as a reference.

Reply:
ok.